### ISSUE TITLE
xcrun error in xcode 15.0

### ISSUE THREAD

**Issue by pgfoster** [xenodium/homebrew-macosrec#1]
I am running Ventura 15.5.2 on apple M1, with Xcode 15.0 (although homebrew specified 14.0).  xcode-select tells me version 2397.

Here are the last few lines of the brew install macosrec output

==> Installing macosrec from xenodium/macosrec
==> swift build -c release --disable-sandbox
Last 15 lines from /Users/<me>/Library/Logs/Homebrew/macosrec/01.swift:
2023-09-28 08:34:22 +0000

swift
build
-c
release
--disable-sandbox

error: terminated(1): /usr/bin/xcrun --sdk macosx --show-sdk-platform-path output:
    xcrun: error: unable to lookup item 'PlatformPath' from command line tools installation
    xcrun: error: unable to lookup item 'PlatformPath' in SDK '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk'

**Comment by xenodium** [comment]
Sorry for the delay. Is this still an issue? Just installed with Xcode 15.

**Comment by pgfoster** [comment]
Sorry, I should have done a bit more digging, and then followed up on this.  I think the problem arose when I installed full xcode on top of my existing command-line tools.  [The fix is described here](https://stackoverflow.com/questions/43418533/running-swift-build-in-terminal-leading-to-platform-path-errors  )
I needed to do
`sudo xcode-select -switch /Applications/Xcode.app/Contents/Developer`
and that sorted it.  So it does not appear to be an issue with macosrec.  Thanks for the nudge, and the useful program.
